theano/configparser.py

```diff
@@ -98,15 +98,17 @@ class change_flags(object):
 
     Useful during tests.
     """
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
         confs = dict()
-        for k in kwargs:
+        args = dict(*args)
```

Comment:
I'm not sure this is clear enough for me. Do you mean keeping *args as a parameter, then make sure it is only 1 element and it is a dict?

If so, this would be more restrictive then accepting anything that get converted to a dict. I don't have a strong preference. I just don't understand what it would bring to be more restrictive here except that the code would be more similar, but would make it harder for people to remember and probably look up more frequently what is accepted.